Packs/CrowdStrikeFalconStreamingV2/Integrations/CrowdStrikeFalconStreamingV2/CrowdStrikeFalconStreamingV2.py

```diff
@@ -405,15 +405,15 @@ async def long_running_loop(
                         except Exception as e:
                             demisto.error(f'Failed storing sample events - {e}')
                     demisto.debug(f'Storing offset {offset_to_store}')
-                    demisto.setIntegrationContext(integration_context)
+                    set_to_integration_context_with_retries(integration_context)
                     last_integration_context_set = datetime.utcnow()
     except Exception as e:
         demisto.error(f'An error occurred in the long running loop: {e}')
     finally:
         # store latest fetched event offset in case the loop crashes and we did not reach the 1 minute to store it
-        integration_context = demisto.getIntegrationContext()
+        integration_context = get_integration_context()
```

Comment:
I think you don't need this call, just pass `{'offset': offset_to_store}` to the set function

Reply:
not really because we also have `sample_events` in there

Reply:
`set_to_latest_integration_context` does that for you

Reply:
done